cmake/UnitTests.cmake

```diff
@@ -9,6 +9,7 @@ list(APPEND DGPU_OPENCL_FAILED_TESTS " ")
 list(APPEND IGPU_OPENCL_FAILED_TESTS " ") 
 list(APPEND IGPU_LEVEL0_FAILED_TESTS " ")
 list(APPEND DGPU_LEVEL0_FAILED_TESTS " ") 
+list(APPEND POCL_LEVEL0_FAILED_TESTS " ") 
```

Comment:
I believe your are testing with the CPU driver still? PoCL-L0 likely fails a lot more tests currently. Best name this POCL_CPU_FAILED_TESTS as we will add also POCL_LEVEL0_FAILED_TESTS when most of the tests pass?